ci/docker-compose.yml

```diff
@@ -13,7 +13,7 @@ services:
     image: "conjur:${TAG}"
     environment:
       DATABASE_URL: postgres://postgres@pg/postgres
-      CONJUR_ADMIN_PASSWORD: admin
+      CONJUR_ADMIN_PASSWORD: ADmin123!!!!
```

Comment:
Can we check if we inject environment variable from Jenkins or other places that will override new compilent password with other one?